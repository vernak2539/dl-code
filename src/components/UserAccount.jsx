import React from 'react';
import userIcon from '../images/user-icon.svg';

const UserIcon = () => <img src={userIcon} />;

export default () => (
	<div className="user-account-container">
		<UserIcon />
		Jane Smith
	</div>
);
