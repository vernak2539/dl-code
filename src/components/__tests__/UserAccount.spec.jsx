import React from 'react';
import { render } from 'enzyme';
import UserAccount from '../UserAccount';

describe('Component > User Account', () => {
	it('should render correctly', () => {
		expect(render(<UserAccount />)).toMatchSnapshot();
	});
});
