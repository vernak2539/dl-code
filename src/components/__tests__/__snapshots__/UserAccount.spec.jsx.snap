// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component > User Account should render correctly 1`] = `
<div
  class="user-account-container"
>
  <img
    src="user-icon.svg"
  >
    
  </img>
  Jane Smith
</div>
`;
